app/src/lib/components/BranchLanePopupMenu.svelte

```diff
@@ -159,7 +159,7 @@
 			kind="solid"
 			on:click={async () => {
 				await branchController.deleteBranch(branch.id);
-				deleteBranchModal.close();
+				close();
```

Comment:
😁 a cheeky unrelated change... if only we could work in multiple branches at the same time